BVLinearGradient/BVLinearGradientLayer.m

```diff
@@ -48,21 +48,44 @@ - (void)setEndPoint:(CGPoint)endPoint
 - (void)display {
     [super display];
 
+    CGSize size = self.bounds.size;
+    if (size.width <= 0 || size.height <= 0) {
+        return;
```

Comment:
will it be blank when the width/height is zero?

Reply:
yes